src/main/groovy/io/saagie/plugin/dataops/models/JobOverride.groovy

```diff
@@ -5,7 +5,7 @@ package io.saagie.plugin.dataops.models
  * This provides a flexible option to set environment specific values.
  * The values thats currently supported are isScheduled, cronScheduling, alerting.
  */
-class JobOverride extends PropertyOverride{
+class JobOverride extends ArtifactPropertyOverride{
```

Comment:
Is that normal that this class is empty ?

Reply:
Well we structered like this thinking that there will be contain in those classes but it didn't happen, i will remove them